src/ResultWriter/DefaultResultWriter.php

```diff
@@ -104,7 +104,7 @@ protected function writeRows(ExportConfig $exportConfig, Iterator $iterator, Csv
         $this->lastRow = $resultRow;
         while ($iterator->valid()) {
             $resultRow = $iterator->current();
-            $csvWriter->writeRow($resultRow);
+            $this->writeRow($resultRow, $csvWriter);
```

Comment:
:facepalm: tu bol preklep, a preto sa prevadzalo kodovanie iba v prvom riadku ... v testoch som mal tiez iba 1 riadok, tak som si to nevsimol. .... a 2h hladam preco to nejde :-D